### DISCUSSION TITLE
How to remove /blog from urls

### DISCUSSION THREAD

**Discussion by dure-jay** [timlrx/tailwind-nextjs-starter-blog#585] [Q&A]
Hello,

I deployed the codebase to Vercel and I'm using blog.domainname.com. The blog is automatically exposed as blog.domainname.com/blog, how can I remove /blog and use / as root instead?

**Comment by dlarroder** [comment]
You can delete `pages/index.js` and then rename `pages/blog.js` to `pages/index.js` so that `/blog` will be your new `/` page

If you want to get rid of the `/blog` in each posts and pages, you can do the following:

1. Move `pages/blog/page/[...slug].js` to `pages/[...slug].js`
2. Move `pages/blog/page/[page].js` to `pages/page/[page].js`
3. Go to `ListLayout.js` and remove `/blog` from `<Link href={'/blog/${slug}'}` on line 66
4. You will also have to fix the import errors from moving the files

Hope this helps!

**Comment by dure-jay** [comment]
Thanks Dale, unfortunately it does not work as expected. The blog can be accessed via /blog and /, I want to get rid of /blog really. I replaced /blog with / in a few files now and the href of the blog post redirects to blog.domain.com/blog-post-name-here <http://domain.com/blog-post-name-here> but it’s not accessible. I think it is related to the router or something, I’m not sure. However, I can still access the blog post via blog.domain.com/blog/blog-post-name-here <http://blog.domain.com/blog/blog-post-name-here>.

**Reply by dlarroder** [reply]
@dure-jay are you sure you followed the steps I wrote? It's just moving files around and it works for me.

<http://domain.com/blog-post-name-here> is not accessible because you haven't moved `pages/blog/page/[...slug].js` to `pages/[...slug].js`

Demo:

https://user-images.githubusercontent.com/52998821/209269365-a9c2d461-ea6d-463b-9ff1-6d7f3cd78ac0.mov

**Comment by dure-jay** [comment]
Thank you Dale, that worked! Have a great day and merry Christmas!
